CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 * @google/generative-ai-admin
 
-demos/palm/web/ @mrayinteractive @aaron-wade @lyleaf @jayjicheng
+demos/palm/web/ @mrayinteractive @aaron-wade @lyleaf @jayjicheng @blownhither
```

Comment:
> This CODEOWNERS file contains errors …
> Unknown owner on line 3: make sure @blownhither exists and has write access to the repository

It'd be good to use more specific access-controls too, e.g. per subdirectory. GitHub is adding everyone on this list to any PR that impacts the directory, so it's hurting the SNR.

Reply:
b/292540968